packages/integrations/stores/launch-darkly/src/launch-darkly.ts

```diff
@@ -0,0 +1,34 @@
+import { type EvaluationContext, OpenFeature } from '@openfeature/server-sdk';
+import { LaunchDarklyProvider } from '@launchdarkly/openfeature-node-server';
+import { type LDOptions } from '@launchdarkly/node-server-sdk';
+import { type ConfigQuery, type Config } from '@configu/sdk';
+import _ from 'lodash';
```

Comment:
Please change the lodash dependency from `lodash` to `lodash-es` and also make sure it's present in the `package.json` dependencies

Reply:
Done.